packages/backend/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export { app } from "./app";
 import { TopicCrudApi } from "./api/topic";
 
+// a list of resources we would like infrastructure generated for
 export const stackResources = [TopicCrudApi];
```

Comment:
We need to import `TopicCrudApi`, but don't need to do so with the function wrapped in `Route({ route: "/blargle" })`?

Reply:
No you're right we do, just haven't gotten that far yet